shuup/admin/modules/products/views/edit.py

```diff
@@ -230,9 +231,9 @@ def get_context_data(self, **kwargs):
         context = super(ProductEditView, self).get_context_data(**kwargs)
         orderability_errors = []
 
+        shop = self.request.shop
```

Comment:
I know you just moved this, but let's fix this, use this instead:
```py
shop = get_shop(self.request)
```

Reply:
Done